pkg/service/service.go

```diff
@@ -219,6 +221,76 @@ func ListWithDetailedStatus(client *occlient.Client, applicationName string) (Se
 	}, nil
 }
 
+// ListOperatorServices lists all operator backed services
+func ListOperatorServices(client *kclient.Client) ([]unstructured.Unstructured, error) {
```

Comment:
This might better suit as a `kclient` struct's method

Reply:
I have kept it here because `func List(client *occlient.Client, applicationName string)` is in the same file for the service catalog based services.

Reply:
Cool, we can keep it here